Added info about manually disabling and enabling cron jobs (#6173)
magento/devdocs: src/guides/v2.2/cloud/configure/setup-cron-jobs.md

1. Add, commit, and push code changes.

   ```bash
   git add -A && git commit -m "cron config updates" && git push origin <branch-name>
   ```

## Update custom cron jobs {#update}

To add, remove, or update a custom cron job, change the configuration in the `crons` section of the `.magento.app.yaml` file for the Integration environment. Then, test the updates in the Integration environment before pushing the changes to the Production and Staging environments.

## Disable cron jobs

In some cases you might want to manually disable cron jobs before you complete maintenance tasks like reindexing or cleaning the cache to prevent performance issues. You can use the ece-tools CLI command `cron:disable` to disable all Magento cron jobs and stop any active cron processes.

{:.procedure}

> To disable cron jobs:

Author: ```suggestion
To disable cron jobs:
```

1. Add, commit, and push code changes.

   ```bash
   git add -A && git commit -m "cron config updates" && git push origin <branch-name>
   ```

## Update custom cron jobs {#update}

To add, remove, or update a custom cron job, change the configuration in the `crons` section of the `.magento.app.yaml` file for the Integration environment. Then, test the updates in the Integration environment before pushing the changes to the Production and Staging environments.

## Disable cron jobs

In some cases you might want to manually disable cron jobs before you complete maintenance tasks like reindexing or cleaning the cache to prevent performance issues. You can use the ece-tools CLI command `cron:disable` to disable all Magento cron jobs and stop any active cron processes.

{:.procedure}
To disable cron jobs when completing maintenance tasks:

> 1. [SSH to your environment]({{ page.baseurl }}/cloud/env/environments-ssh.html#ssh).

Reviewer: Use SSH to log in to ....

> 1. [SSH to your environment]({{ page.baseurl }}/cloud/env/environments-ssh.html#ssh).

Author: ```suggestion
1. Use [SSH to log in to your environment]({{ page.baseurl }}/cloud/env/environments-ssh.html#ssh).
```

1. Disable cron jobs and stop active cron processes.

   ```shell
   ./vendor/bin/ece-tools cron:disable
   ```

> 1. Complete the required task.

Reviewer: This seems vague. The step is to complete the task, but is a task presented after running cron:disable? Or is the task waiting for something to complete? If it is waiting, then would be better to warn that stopping active process may take time.

1. After the task completes, re-enable the cron jobs.

   ```shell
   ./vendor/bin/ece-tools cron:enable
   ```

## Troubleshooting cron jobs

> Magento has updated the {{site.data.var.ece}} package to optimize cron processing on the {{site.data.var.ece}} platform and to fix cron-related issues. If you are having problems with cron processing, make sure that your project is using the most current version of the ece-tools package. See [Upgrades and patches]({{ page.baseurl }}/cloud/project/project-upgrade-parent.html).

Reviewer: ```suggestion
Magento has updated the {{site.data.var.ece}} package to optimize cron processing on the {{site.data.var.ece}} platform and to fix cron-related issues. If you encounter problems with cron processing, make sure that your project is using the most current version of the ece-tools package. See [Upgrades and patches]({{ page.baseurl }}/cloud/project/project-upgrade-parent.html).
```

1. Add, commit, and push code changes.

   ```bash
   git add -A && git commit -m "cron config updates" && git push origin <branch-name>
   ```

## Update custom cron jobs {#update}

To add, remove, or update a custom cron job, change the configuration in the `crons` section of the `.magento.app.yaml` file for the Integration environment. Then, test the updates in the Integration environment before pushing the changes to the Production and Staging environments.

## Disable cron jobs

In some cases you might want to manually disable cron jobs before you complete maintenance tasks like reindexing or cleaning the cache to prevent performance issues. You can use the ece-tools CLI command `cron:disable` to disable all Magento cron jobs and stop any active cron processes.

{:.procedure}
To disable cron jobs:

1. Use [SSH to log in to your environment]({{ page.baseurl }}/cloud/env/environments-ssh.html#ssh).

1. Disable cron jobs and stop active cron processes.

   ```shell
   ./vendor/bin/ece-tools cron:disable
   ```
   

> After you complete the maintenance task, ensure that you enable the cron jobs again.

Author: ```suggestion
After you complete any required maintenance tasks, ensure that you enable the cron jobs again.
```

> After you complete any required maintenance tasks, ensure that you enable the cron jobs again.

Reviewer: It makes sense that this is a step, too, now that it is more clear.

> After you complete any required maintenance tasks, ensure that you enable the cron jobs again.

Author: ```suggestion
1. After you complete any required maintenance tasks, ensure that you enable the cron jobs again.
```

```shell

> ./vendor/bin/ece-tools cron:disable

Author: ```suggestion
./vendor/bin/ece-tools cron:enable
```

```

## Troubleshooting cron jobs

> Magento has updated the {{site.data.var.ece}} package to optimize cron processing on the {{site.data.var.ece}} platform and to fix cron-related issues. If you encounter problems with cron processing, make sure that your project is using the most current version of the ece-tools package. See [Upgrades and patches]({{ page.baseurl }}/cloud/project/project-upgrade-parent.html).

Author: ```suggestion
Magento has updated the {{site.data.var.ece}} package to optimize cron processing on the {{site.data.var.ece}} platform and to fix cron-related issues. If you encounter problems with cron processing, make sure that your project is using the most current version of the {{ site.data.var.ct }} package. See [Upgrades and patches]({{ page.baseurl }}/cloud/project/project-upgrade-parent.html).
```
